File: Actividades Sena/Clases/mayo2024/31_05_24/sistemaDeEstudiantes.rb
puts "Numeros de estudiantes"
e = gets.chomp.to_i
for i in 1..e
  puts "Ingrese su primera nota"
  num1 = gets.chomp.to_f
  puts "Ingrese su segunda nota"
  num2 = gets.chomp.to_f
  puts "Ingrese su tercera nota"
  num3 = gets.chomp.to_f
  prom = (num1 + num2 + num3) / 3
  if prom > 6
    puts "Aprobaste la asignatura con #{prom}"
  else
    puts "Reprobaste la asignatura con #{prom}"
  end

File: Actividades Sena/Clases/mayo2024/31_05_24/sistemaDeEstudiantes.dart
import 'dart:io';

void main() {
  print('Numeros de estudiantes');
  int e = int.parse(stdin.readLineSync()!);
  for (int i = 1; i <= e; i++) {
    print('Ingrese su primera nota');
    double num1 = double.parse(stdin.readLineSync()!);
    print('Ingrese su segunda nota');
    double num2 = double.parse(stdin.readLineSync()!);
    print('Ingrese su tercera nota');
    double num3 = double.parse(stdin.readLineSync()!);
    double prom = (num1 + num2 + num3) / 3;
    if (prom > 6) {
      print('Aprobaste la asignatura con $prom');
    } else {
      print('Reprobaste la asignatura con $prom');
    }
  }
}
